### PR TITLE
V22F-907 Save password locally with delete button

### PR DESCRIPTION
**Pull request by LucasDissie**

### CODE REVIEW

File: speed-dreams/source-2.2.3/src/modules/userinterface/legacymenu/mainscreens/DatabaseConnectionManager.cpp [hunk 1]
```diff
@@ -278,6 +282,22 @@ void ClearPassword(void* p_scrHandle, int p_passwordControl)
     GfuiEditboxSetString(p_scrHandle, p_passwordControl, "");
 }
 
+/// Delete the password in from the settings and from the XML file
```
Comment (larsdekwant) [inline speed-dreams/source-2.2.3/src/modules/userinterface/legacymenu/mainscreens/DatabaseConnectionManager.cpp:285]:
```suggestion
/// @brief Delete the password in from the settings and from the XML file
```

**Review by larsdekwant — CHANGES_REQUESTED** [review]
Small comment issue and refactor to clean up code.

**Review by larsdekwant — APPROVED** [review]

File: speed-dreams/source-2.2.3/data/data/menu/DatabaseSettingsMenu.xml [hunk 2]
```diff
@@ -148,6 +148,29 @@
             <attstr name="enabled image" val="data/img/button-left.png"/>
             <attstr name="pushed image" val="data/img/button-left-pushed.png"/>
         </section>
+		
+		<!-- Delete password button -->
```
Comment (suppergerrie2) [inline speed-dreams/source-2.2.3/data/data/menu/DatabaseSettingsMenu.xml:152]: Use spaces instead of tabs

**Review by suppergerrie2 — CHANGES_REQUESTED** [review]
Add changes to code changes document

**Review by suppergerrie2 — APPROVED** [review]

**Pull request merged**